editor/src/EditorStartup.cpp

```diff
@@ -273,6 +272,7 @@ void EditorStartup::stopGame()
             updateSceneManagerFunct();
 
             m_engine->inputManager.processInput();
+            m_engine->soundSystem.update();
```

Comment:
There never is a situation when we can play sound in the editor without simulating the game.
I believe it is unnecessary to update the sound system here.

```suggestion
```

Reply:
same than before